expo-maps/ios/MapView.swift

```diff
@@ -0,0 +1,8 @@
+protocol MapView: UIView {
+  var mapView: UIView? { get }
+  var authenticated: Bool { get }
```

Comment:
For boolean properties it's usually better to name it like `isAuthenticated` rather than `authenticated`